cmake/FindTarantool.cmake

```diff
@@ -24,7 +24,7 @@ if(TARANTOOL_INCLUDE_DIR)
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(TARANTOOL
+find_package_handle_standard_args(Tarantool
```

Comment:
But the result will anyway available in the `TARANTOOL_FOUND` variable? I don't understand how it works. The argument is uppercased? Variable names are case insensetive?

For myself I just suppressed the warning (but if we'll understood why we can use the `Tarantool` argument here, it would be better, of course). See https://github.com/tarantool/smtp/pull/31.

Reply:
I found this info related to a warning:
- https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindPackageHandleStandardArgs.cmake#L435-444
- https://gitlab.kitware.com/cmake/cmake/-/issues/16884

Reply:
It is a bit annoying that we have same warnings in many projects.
It was already fixed in DDL - https://github.com/tarantool/ddl/commit/c17ebf8015c18ece10858e95674c30666c4330c3

Reply:
The argument is uppercased, [I see it now](https://gitlab.kitware.com/cmake/cmake/-/blob/5bd94332c760a6f861b411273921b204076ab9a6/Modules/FindPackageHandleStandardArgs.cmake#L470). Okay. (The weird thing here is that it is documented in the usual CMake style: tell nobody about that.)

Reply:
Besides `*_FOUND` variable this line affects handling `set(Tarantool_FIND_REQUIRED ON)`

Reply:
Thanks for the noticing! I filed PR #20 with the fix.